tests/subsys/usb/desc_sections/src/desc_sections.c

```diff
@@ -145,6 +145,7 @@ static void test_desc_sections(void)
 	zassert_equal((int)__usb_descriptor_end - (int)__usb_descriptor_start,
 		      119, NULL);
 
+	zassert_equal(__usb_data_end - __usb_data_start, 2, NULL);
```

Comment:
please describe it, it is confusing at first glance. btw, can we fix the magic 119 and 88?

Reply:
I fixed one magic, another requires sizeof() of private structures from another file, so I left it as is.